service/zpages.go

```diff
@@ -26,17 +26,20 @@ import (
 )
 
 const (
-	servicezPath   = "servicez"
-	pipelinezPath  = "pipelinez"
-	extensionzPath = "extensionz"
-	featurezPath   = "featurez"
+	zPipelineName  = "zpipelinename"
+	zComponentName = "zcomponentname"
+	zComponentKind = "zcomponentkind"
+	zService       = "servicez"
+	zPipeline      = "pipelinez"
+	zExtension     = "extensionz"
+	zFeature       = "featurez"
```

Comment:
I don't know enough about zpages - Can we rename the values to use the same format? Or will this be a breaking change for users?
```suggestion
	zPipelineName  = "zpipelinename"
	zComponentName = "zcomponentname"
	zComponentKind = "zcomponentkind"
	zService       = "zservice"
	zPipeline      = "zpipeline"
	zExtension     = "zextension"
	zFeature       = "zfeature"
```

Reply:
the `zService` (a.k.a the one that finish in z) are paths (so will be breaking change). The other are used as URL params so you can change. So if you want to consolidate, you do it the other way around.